requirements.txt

```diff
@@ -10,8 +10,11 @@ restless==2.0.1
 # PyYAML is for contact loading
 PyYAML==3.11
 
-# deployed/hosted with gunicorn
-gunicorn
-dj-static
+# deployed/hosted with waitress
+waitress==0.8.9
+dj-static==0.0.6
```

Comment:
dj-static was already a requirement. It handles static files in a WSGI context. Beyond that -- I'm just following instructions, and don't know much more.